tests/spec/core/ui-spec.js

```diff
@@ -89,4 +89,40 @@ describe("Core - UI", () => {
       expect(dfnFoo.getAttribute("href")).toBe("#dfn-foo");
     });
   });
+
+  it("shows a list of definitions and labels definitions that are unused and/or exported", async () => {
+    const body = `
+      <p>
+        <dfn>foo</dfn>
+        <dfn data-export="">bar()</dfn>
+        <dfn data-export="">foobar</dfn>
+        <dfn>baz</dfn>
+      </p>
+      <p>References to <a>Foo</a> and <a>foobar</a>.</p>
+    `;
+
+    const ops = makeStandardOps(null, body);
+    const doc = await makeRSDoc(ops);
+
+    // open list and wait for it to load
+    const dfnListButton = doc.getElementById("respec-button-definitions");
+    dfnListButton.click();
+    await new Promise(resolve => setTimeout(resolve));
+
+    const dfns = doc.querySelectorAll("ul.respec-dfn-list li");
+    expect(dfns.length).toBe(4);
+
+    const [dfnBar, dfnBaz, dfnFoo, dfnFoobar] = dfns;
+
+    expect(dfnBar.innerText.includes("(exported) (unused)")).toBeTrue();
```

Comment:
Nit:
```suggestion
    expect(dfnBar.textContent).toContain("(exported) (unused)");
```

Reply:
@sidvishnoi ah - was looking for something like `toContain` and didn't find it in the docs. thanks!

could you explain why you prefer `textContent` over `innerText`?

Reply:
`textContent` vs `innerText`: https://stackoverflow.com/a/35213639
tl;dr: textContent is more performant, innerText capabilities are not needed here. We use textContent in most (all?) tests, so it's also consistent.

Reply:
fixed, thanks for explaining!